languages/en_US.po

```diff
@@ -3,7 +3,7 @@ msgstr ""
 "Project-Id-Version: gppro-entry-content\n"
 "POT-Creation-Date: 2014-03-22 14:56-0500\n"
 "PO-Revision-Date: 2014-03-22 14:56-0500\n"
-"Last-Translator: Andrew Norcross <andrew@andrewnorcross.com>\n"
+"Last-Translator: Reaktiv Studios <andrew@andrewnorcross.com>\n"
```

Comment:
Should we change the email address on this?

Reply:
what do you think it should be? I thought about dev@reaktivstudios.com but I'm not sure that is ideal. Maybe support@?

Reply:
@NicktheGeek I discussed this with @jjeaton, and he said just remove the email address.